catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -43,12 +48,20 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
 
     Map<String, String> aliases =
         (Map<String, String>) arguments.getOrDefault("aliases", new HashMap<>());
-
+    String attributeString =
+        arguments.get("columnOrder") != null ? (String) arguments.get("columnOrder") : "";
+    List<String> attributes = Arrays.asList((attributeString).split(","));
+    List<AttributeDescriptor> allAttributes =
+        new ArrayList<AttributeDescriptor>(metacard.getMetacardType().getAttributeDescriptors());
+    List<AttributeDescriptor> descriptors =
+        CollectionUtils.isEmpty(attributes)
+            ? allAttributes
```

Comment:
According to the documentation for CollectionUtils.isEmpty, it returns true on null. Do we want to return null here?

Reply:
`attributes` will never be null because we initialize it with `Arrays.asList((attributeString).split(","))`, which will return an empty list when given an empty string.  `attributeString` defaults to the empty string if no `columnOrder` argument is found.